### ISSUE TITLE
Badges position issue in email on iOS

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#11005]
**Build:**
Production (Train 217)

**Preconditions:**
The user doesn’t have a Firefox account.

**Affected platforms:**
iOS 14.8

**Steps to reproduce:**

1. Launch Firefox browser.
1. Go to Hamburger menu and tap on “Settings“.
1. Tap on “Sign in to Sync” and then on “Use email instead”.
1. Fill in valid credentials and tap “Create account” button.
1. Verify your inbox for the verification code.
1. Copy the verification code and paste it in the field.
1. Click the “Verify” button.
1. Observe the email received “Account verified...”

**Actual result:** 
The badges are displayed one below the other.

**Expected result:**
The badges are displayed one near the other.

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-4197)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/secure/attachment/18345/IMG-0843.jpg">IMG-0843.jpg</a> | <a href="https://mozilla-hub.atlassian.net/secure/attachment/17878/iOS+badges+position+issue.PNG">iOS badges position issue.PNG</a> | <a href="https://mozilla-hub.atlassian.net/secure/attachment/18344/Screen+Shot+2021-12-03+at+11.13.28+AM.png">Screen Shot 2021-12-03 at 11.13.28 AM.png</a>

**Comment by data-sync-user** [comment]
➤ Lisa Chan commented:

Bianca Oltean , I am having trouble reproducing the issue, can you let us know what device it was tested on? Thanks!

**Comment by data-sync-user** [comment]
➤ Bianca Oltean commented:

Hi Lisa Chan The test was made on iphone 11. I made the test again today on stage [Train 220] and 
I get the same result.

**Comment by data-sync-user** [comment]
➤ Lisa Chan commented:

Thanks for confirming the device, Bianca Oltean! Can you also let us know whether you get the same results with the new email templates?

**Comment by data-sync-user** [comment]
➤ Bianca Oltean commented:

Lisa Chan As far as I know the new email templates are already on stage [Train 220] so the test was made with the new email templates.

Lauren Zugai Am I correct?

**Comment by data-sync-user** [comment]
➤ Lisa Chan commented:

Bianca Oltean , I was able to regroup with Lauren Zugai and determine that this issue is occurring on an old email template (see below for screenshots). In the new email templates, the footer text should now be three lines due to the increase in font-size, and the color of the blue button has been updated based on the colors in FxA’s design guide.

Lauren was also able to confirm that the new email templates should have been in staging at the time of testing, but only for accounts using an @mozilla.com email. Other emails are still receiving the old templates.

As this issue is only affecting old email templates, we will move forward with marking this as a won't do. However, if this issue occurs when testing the new email templates, we will re-open the issue and move it into the emails tasks and bugs epic.

Old template:

!IMG-0843.jpg|width=768,height=720!

New template:

!Screen Shot 2021-12-03 at 11.13.28 AM.png|width=495,height=403!
